src/Main.java

```diff
@@ -6,5 +6,101 @@ public static void main(String[] args) {
         // to see how IntelliJ IDEA suggests fixing it.
         System.out.println("Hello and welcome!");
         System.out.println("variable-Lesson2");
+        System.out.println("Задача 1");
+        int A = 4000;
```

Comment:
Названия переменных - первая буква в нижнем регистре
```suggestion
        int a = 4000;
```